testdata/project-v2/config/default/manager_auth_proxy_patch.yaml

```diff
@@ -1,5 +1,5 @@
-# This patch inject a sidecar container which is a HTTP proxy for the controller manager,
-# it performs RBAC authorization against the Kubernetes API using SubjectAccessReviews.
+# This patch inject a sidecar container which is a HTTP proxy for the 
+# controller manager, it performs RBAC authorization against the Kubernetes API using SubjectAccessReviews.
```

Comment:
Same as above

Reply:
This line is in the correct size. 
The lint issue was in the line where the const is defined.